tests/llmobs/test_llmobs.py

```diff
@@ -628,3 +628,27 @@ def test_trace_id_propagation_with_non_llm_parent(llmobs, llmobs_events):
     # LLMObs trace IDs should be different from APM trace ID
     assert first_child_event["trace_id"] != first_child_event["_dd"]["apm_trace_id"]
     assert second_child_event["trace_id"] != second_child_event["_dd"]["apm_trace_id"]
+
+
+@pytest.mark.parametrize("llmobs_env", [{"DD_APM_TRACING_ENABLED": "false"}])
+def test_apm_traces_dropped_when_disabled(llmobs, llmobs_events, tracer, llmobs_env):
+    from tests.utils import DummyWriter
```

Comment:
Should we move this import to top of file?

Reply:
no pref